frontend/src/app/api/auth/signin/route.ts

```
@@ -0,0 +1,55 @@
+        });
+
+        const drfResponseJson = await drfResponse.json();
+        if (drfResponse.ok) {
+            // extract token from response
+            const {access, refresh} = drfResponseJson.data;
+            const cookieStore = await cookies();
+            cookieStore.set("access", access, {path: "/", sameSite: "strict", httpOnly: true, secure: true});
+            cookieStore.set("refresh", refresh, {path: "/", sameSite: "strict", httpOnly: true, secure: true});
+
+            // remove token from response and add redirect url
+            delete drfResponseJson.data;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)